### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by lilithxxx**
Changes in this pull request:

-Linked python3 official documentation
-
-

### CODE REVIEW

**Comment by lilithxxx** [comment]
#1196 I referenced this issue before and want to update README.md file
@ekohl

**Comment by magul** [comment]
Hi @lilithxxx What do You think about moving it to the bottom ot this chapter? I feel that it would be strange to be redirected to other resource just at the beginning here.

What do You think?

**Comment by lilithxxx** [comment]
I feel they should be free to choose whether to study the full language before going on with the tutorial as it provides a thorough and complete study.
What do you think of that? @magul

**Comment by magul** [comment]
I just feel that main reason of existence of this tutorial is to take everybody in weekend-long journey from total ignorance into fully working blog somewhere on the Internet. The purpose of this chapter is to give them bunch of information how Python works (and if they feel, that they want to know more, point them to appropriate resources). Tey will still have a choice at the end of this chapter, but it will not make them to loose a focus on Django Girls tutorial even before they start to learn basics of Python.

But maybe we should wait for others thoughts..

**Comment by ekohl** [comment]
I agree with @magul that this is a bit of a shortened tutorial that should be enough for the workshop. The bottom might be better suited.

@magul Is this somewhere is a weekend-long workshop? I've only done it in a single day (in 2 different locations). Makes it even harder. In the 5 times I've been a mentor there have been 2 students who finished it.

**Comment by magul** [comment]
@ekohl we have done it in two days of weekend in Poznań. we had time and it's a lot more comfortable to do it a little bit slower. And even if You would done it in 2 days, it doesn't mean that a lot more of participants will end it the workshop.

**Comment by ekohl** [comment]
@magul I think I do agree with you - many only get halfway there which is why I always try to stress that the instructions should be short and brief.

**Comment by lilithxxx** [comment]
I have edited the PR and moved the link to the bottom of the chapter.

What do you think?

File: en/python_introduction/README.md [hunk 1]
```diff
@@ -1003,6 +1002,9 @@ Note that the second of these two numbers is not included in the list that is ou
 
 That's it. __You totally rock!__ This was a tricky chapter, so you should feel proud of yourself. We're definitely proud of you for making it this far!
 
+For official and full python tutorial visit https://docs.python.org/3/tutorial/. This will give you a more thorough and complete study of the language.Cheers :)
```
Comment (magul) [inline en/python_introduction/README.md]: There's space missing after dot.

**Pull request merged**